src/CSET/operators/plot.py

```diff
@@ -421,8 +421,8 @@ def _plot_and_save_line_series(
 
     Parameters
     ----------
-    cube: Cube or CubeList
-        Cube or CubeList containing the cubes to plot on the y-axis.
+    cubes: Cubes or CubeList
+        Cubes or CubeList containing the cubes to plot on the y-axis.
```

Comment:
```suggestion
    cubes: Cube or CubeList
        Cube or CubeList containing the cubes to plot on the y-axis.
```
The data type is still (possibly) an `iris.cube.Cube`.